package.json

```diff
@@ -85,6 +85,7 @@
     "@mui/material": "^6.1.0",
     "algoliasearch": "^5.2.4",
     "axios": "^1.7.7",
+    "canvas": "^2.11.2",
```

Comment:
Is `canvas` a new peer dependency requirement? I'm not seeing where this used.

Reply:
The canvas package is essential for resolving HTMLCanvasElement.prototype.getContext errors in our test environment. Without this package, running yarn test would result in numerous errors, as the canvas-related functionalities required for simulations would be missing.

Reply:
Okay. It's not listed as a peer dependency of `lottie-react`, although I do see `jest-canvas-mock`. Is the `canvas` package related to that? Just trying to understand what depends on this new package, as it was not needed before.

Reply:
The react-lottie library, which is used for rendering Lottie animations in React, does rely on the canvas element through the bodymovin library. Bodymovin (part of lottie-web) uses canvas or SVG as its renderer, depending on the animation's complexity and settings. When adding react-lottie, this canvas dependency can require additional setup in testing environments like Jest or Vitest, particularly for mock or DOM rendering support.